configuration/persistence.md

```diff
@@ -65,14 +65,75 @@ The following strategies are defined internally and may be used in place of `str
 openHAB uses [Quartz](https://www.quartz-scheduler.org/documentation) for time-related cron events.
 See the [Rules article]({{base}}/configuration/rules-dsl.html#time-based-triggers) for more information.
 
+### Filters
+
+This (optional) section allows you to name and define one or more `Filters`.
+The syntax is as follows:
+
+```
+Filters {
+  <filterName1> : <filter definition>
+  <filterName2> : <filter definition>
+  ...
+}
+```
+
+Each filter definition starts with a single character that identifies the filter type.
+
+#### Threshold Filter (`>`)
+
+The threshold filter allows to discard values that are close to the last persisted value.
+It can only be used with `Number` items and has two modes: "relative" and "absolute".
+
+In "relative" mode the threshold is calculated as percentage of the last persisted value.
+This mode is selected by a `%` character and a decimal value (e.g. `> % 10` for "at least 10% deviation").
+
+In "absolute" mode the threshold is calculated by absolute difference to the last persisted value.
+This mode is selected by a decimal value with optional unit (e.g. `> 5 mA` for "at least 5 mA deviation").
+For plain `Number` items the value can be omitted.
+
+```
+Filters {
+  fiveMilliAmps : > 5 mA
+  tenPercent : > % 10
+}
+```
+
+#### Time Filter (`T`)
+
+The time filter allows to discard values that arrive within the given time after a previous value (e.g. when using the `everyUpdate` strategy).
+It can be used with every item type.
+The filter definition consists of an integer value followed by a unit.
+The allowed units are `s` (seconds), `m` (minutes), `h` hours and `d` days.
```

Comment:
Can you mix units, e.g. `1 m 30 s` or would one need to use `90 s` for that case?